src/cli_driver.rs

```diff
@@ -457,6 +462,22 @@ impl ProcessingSession {
             }
         }
 
+        if let Some(dir) = args.value_of("outdir") {
+
+            output_path = {
+                let tmp = PathBuf::from(dir);
+                match tmp.extension() {
+                    Some(_) => {
+                        tt_warning!(status, "file extention in outdir; only the directory will be used");
```

Comment:
extention => extension
Maybe also `outdir` => `` `-o/--outdir` ``, so it's clear that outdir means the command line option?